lib/config.coffee

```diff
@@ -4,6 +4,22 @@ module.exports =
 
     root = document.documentElement
 
+
+    # Font Size
+    setFontSize = (fontSize) ->
+      _fontSize = atom.config.get('one-dark-ui.fontSize')
```

Comment:
:+1:  Actually, just realized I don't even need the `atom.config.get` since it gets passed in already.